docs/video-miners/core-concepts/roles-and-responsibilities.md

```diff
@@ -1,58 +1,67 @@
-and transcoding capacity (affected by bandwidth and computational capacity).
-Since a broadcaster selects orchestrators to send work to based on these
-factors, an orchestrator's earning potential depends on its ability to perform
-well in each of these areas.
-
-Transcoders perform work on the network. They provide computational capacity to
-orchestrators with hardware resources. An orchestrator can also be a transcoder
-itself, but also has the option of outsourcing transcoding to one or many
-separate transcoders similar to how PoW mining pools outsource PoW hashing to
-one or many separate miners. Orchestrators that outsource transcoding to one or
-many separate transcoders are commonly referred to as
-[transcoding pools](/video-miners/core-concepts/pools).
-
-## Transcoding
-
-Transcoding is the process of taking an input source video in one format and
-bitrate, and converting it to many output rendition videos with different
-formats and bitrates such that the video can be played on the majority of the
-devices in the world at any internet connection speed. Additional details on
-transcoding can be found in
-[this blog post](https://livepeer.com/blog/intro-to-transcoding).
-
-In the Livepeer network, broadcasters and video miners establish a price for
-transcoding on the open market. Once a broadcaster has selected one or many
-video miners to work with it will send segments of a video stream to the video
-miners with micropayments based on the agreed upon price. The video miners then
-return transcoded results for each segment to the broadcasters.
+- [Orchestrators](/video-miners/core-concepts/roles-and-responsibilities#about-orchestrators) are responsible for receiving video from broadcasters and returning transcoded results.
+- [Transcoders](/video-miners/core-concepts/roles-and-responsibilities#about-transcoders) are responsible for transcoding video.
+
+### About Orchestrators 
+
+An orchestrator coordinates work on the network. They earn LPT rewards by staking their own LPT, attracting the stake of delegators by sharing earnings, and earning ETH fees for providing high-quality transcoding services to broadcasters. 
+
+The quality of transcoding services is influenced by a combination of factors, including but not limited to: 
+
+- Economic security (the stake of the orchestrator and its delegators which can be slashed if the work is performed incorrectly), 
+
+- Geographic coverage and transcoding capacity (affected by bandwidth and computational capacity).
+
+A broadcaster selects orchestrators to send work to based on these factors. Hence, an orchestrator's earning potential depends on their performance in these areas.
+
+An orchestrator can perform transcoding with the additional capability of outsourcing transcoding to one or many transcoders commonly referred to as a [transcoding pool](/video-miners/core-concepts/pools). They are similar to how PoW mining pools outsource PoW hashing to one or many separate miners. 
```

Comment:
Nuance here:
```
An orchestrator can perform transcoding with the additional capability of outsourcing transcoding to one or many transcoders
```

Technically speaking, they'd be running a combined orchestrator / transcoder, and the transcoder would be doing the transcoding.